pandas/core/internals/construction.py

```diff
@@ -980,7 +980,7 @@ def convert_object_array(
     ----------
     content: List[np.ndarray]
     dtype: np.dtype or ExtensionDtype
-    use_nullable_dtypes: Controls if nullable dtypes are returned.
+    dtype_backend: Controls if nullable dtypes are returned.
```

Comment:
Probably ok, but maybe we can update this description.

Reply:
Updated